lib/controllers/logout.js

```diff
@@ -14,11 +14,17 @@ var middleware = require('../middleware');
  */
 module.exports = function(req, res) {
   var config = req.app.get('stormpathConfig');
+  var accepts = req.accepts([ 'json' ]);
```

Comment:
`login.js` uses this style, but I will change them both to not use spaces. I also decided to add `html` to the list to be consistent and be clear that this route supports both html and json.